<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="623ms"
		durationms="623"
		type="root"
		rid="53f2913f4e90496"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="black"
		rxversion="9.0.0.0"
		host="DESKTOP-Q3681E5"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1366x768"
		timestamp="4/17/2019 9:45:28 AM"
		timestampiso="2019-04-17T02:45:28"
		endtime="4/17/2019 9:45:29 AM"
		timeoutfactor="1">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:00.622"
			timeRelativeToTestModuleStartTime="00:00.623"
			timeWallClock="09:45:29"
			time="00:00.622"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
